code/modules/client/preference/loadout/loadout_neck.dm

```diff
@@ -0,0 +1,206 @@
+/*
+######################################################################################
+##																					##
+##								IMPORTANT README									##
+##																					##
+##	  Changing any /datum/gear typepaths --WILL-- break people's loadouts.			##
+##	The typepaths are stored directly in the `characters.gear` column of the DB.	##
+##		Please inform the server host if you wish to modify any of these.			##
```

Comment:
Honestly, if this needs aa's attention, might be worth giving him codeowner on these files in the future.

(Not a change you need to make, just a comment)